tools/halide_image_io.h

```diff
@@ -71,7 +71,7 @@ typedef bool (*CheckFunc)(bool condition, const char *msg);
 inline bool CheckFail(bool condition, const char *msg) {
     if (!condition) {
         fprintf(stderr, "%s\n", msg);
-        exit(-1);
+        abort();
```

Comment:
Why is abort() better than exit(-1) here?

Reply:
So that you can drop in with gdb and figure out what the call stack was at the point of error. With exit(-1) you don't get that.

Reply:
Fair. (FWIW, CTest treats abort() as 'unrecoverable failure' that it refuses to treat as 'ordinary test failure'.)

Reply:
@steven-johnson -- the error tests get around that by catching the abort signal and translating them to exit(1)'s. See https://github.com/halide/Halide/blob/master/test/common/expect_abort.cpp